src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -748,6 +680,150 @@ bool emberAfColorControlClusterStepHueCallback(chip::app::Command * commandObj,
     return true;
 }
 
+/** @brief Move To Hue And Saturation
+ *
+ *
+ *
+ * @param hue   Ver.: always
+ * @param saturation   Ver.: always
+ * @param transitionTime   Ver.: always
+ */
+bool emberAfColorControlClusterMoveToHueAndSaturationCallback(chip::app::Command * commandObj, uint8_t hue, uint8_t saturation,
+                                                              uint16_t transitionTime, uint8_t optionsMask, uint8_t optionsOverride)
+{
+    return moveToHueAndSaturation(static_cast<uint16_t>(hue), saturation, transitionTime, optionsMask, optionsOverride, false);
+}
+
+bool emberAfColorControlClusterMoveHueCallback(chip::app::Command * commandObj, uint8_t moveMode, uint8_t rate, uint8_t optionsMask,
+                                               uint8_t optionsOverride)
+{
+    return moveHue(moveMode, static_cast<uint16_t>(rate), optionsMask, optionsOverride, false);
+}
+
+bool emberAfColorControlClusterMoveSaturationCallback(chip::app::Command * commandObj, uint8_t moveMode, uint8_t rate,
```

Comment:
This function did not change at all, so might have been nice to leave it in the old place in the file, but ok.

Reply:
Moved it back up where it was. Total diff should be cleaner now